modules/apps/commerce/commerce-address-web/src/main/java/com/liferay/commerce/address/web/internal/display/context/CommerceChannelAccountEntryRelDisplayContext.java

```diff
@@ -223,10 +233,21 @@ public int getType() {
 		return _type;
 	}
 
-	public boolean hasPermission(String actionId) throws PortalException {
```

Comment:
this is a public method, so even if we are always using the UPDATE action, I'll keep the parameter instead, so we don't change the method signature